styx-api-service/src/main/java/com/spotify/styx/api/BackfillResource.java

```diff
@@ -495,4 +511,9 @@ private RunStateData getRunStateData(Backfill backfill,
     return ReplayEvents.getBackfillRunStateData(wfi, storage, backfill.id())
         .orElse(RunStateData.create(wfi, UNKNOWN, StateData.zero()));
   }
+
+  @FunctionalInterface
+  interface CurrentTimer{
```

Comment:
We can use https://github.com/spotify/styx/blob/master/styx-common/src/main/java/com/spotify/styx/util/Time.java instead.